File: Ex5.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Введите число: ");
        double osnovanie = scanner.nextDouble();
        System.out.print("Введите степень: ");
        double stepen1 = scanner.nextDouble();
        System.out.print("Введите новую степень: ");
        double stepen2 = scanner.nextDouble();

        System.out.printf("Новая степень: %.2f%n", stepen2);
    }
}

File: Ex5.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double osnovanie, stepen1, stepen2;

    cout << "Введите число: ";
    cin >> osnovanie;
    cout << "Введите степень: ";
    cin >> stepen1;
    cout << "Введите новую степень: ";
    cin >> stepen2;

    cout << fixed << setprecision(2) << "Новая степень: " << stepen2 << endl;

    return 0;
}
